src/contract_wrappers/modules/checkpoint/voting/advanced_p_l_c_r_voting_checkpoint_wrapper/index.ts

```diff
@@ -0,0 +1,74 @@
+/* istanbul ignore file */
+import { AdvancedPLCRVotingCheckpointEventArgs_3_1_0 } from '@polymathnetwork/abi-wrappers';
+import { AdvancedPLCRVotingCheckpoint_3_1_0, isAdvancedPLCRVotingCheckpoint_3_1_0 } from './3.1.0';
+
+import Common, {
+  isAdvancedPLCRVotingCheckpoint,
+  BallotParams,
+  StatutoryBallotParams,
+  CustomStatutoryBallotParams,
+  CustomCumulativeBallotParams,
+  CumulativeBallotParams,
+  CustomCumulativeBallotWithExemptionParams,
+  CumulativeBallotWithExemptionParams,
+  StatutoryBallotWithExemptionParams,
+  CustomStatutoryBallotWithExemptionParams,
+  CommitVoteParams,
+  RevealVoteParams,
+  CancelBallotParams,
+  ChangeBallotExemptedVotersListParams,
+  ChangeBallotExemptedVotersListMultiParams,
+  ChangeDefaultExemptedVotersListParams,
+  ChangeDefaultExemptedVotersListMultiParams,
+  CheckpointIdParams,
+  BallotIdParams,
+  VoteTokenCountParams,
+} from './common';
+import { ContractVersion } from '../../../../../types';
+
+export type AdvancedPLCRVotingCheckpointEventArgs = AdvancedPLCRVotingCheckpointEventArgs_3_1_0;
+
+export {
+  AdvancedPLCRVotingCheckpointEvents_3_1_0 as AdvancedPLCRVotingCheckpointEvents,
+  AdvancedPLCRVotingCheckpointStatutoryBallotCreatedEventArgs_3_1_0 as AdvancedPLCRVotingCheckpointStatutoryBallotCreatedEventArgs,
+  AdvancedPLCRVotingCheckpointCumulativeBallotCreatedEventArgs_3_1_0 as AdvancedPLCRVotingCheckpointCumulativeBallotCreatedEventArgs,
+  AdvancedPLCRVotingCheckpointVotersExemptedEventArgs_3_1_0 as AdvancedPLCRVotingCheckpointVotersExemptedEventArgs,
+  AdvancedPLCRVotingCheckpointVoteCommitEventArgs_3_1_0 as AdvancedPLCRVotingCheckpointVoteCommitEventArgs,
+  AdvancedPLCRVotingCheckpointVoteRevealedEventArgs_3_1_0 as AdvancedPLCRVotingCheckpointVoteRevealedEventArgs,
+  AdvancedPLCRVotingCheckpointBallotCancelledEventArgs_3_1_0 as AdvancedPLCRVotingCheckpointBallotCancelledEventArgs,
+  AdvancedPLCRVotingCheckpointChangedBallotExemptedVotersListEventArgs_3_1_0 as AdvancedPLCRVotingCheckpointChangedBallotExemptedVotersListEventArgs,
+  AdvancedPLCRVotingCheckpointChangedDefaultExemptedVotersListEventArgs_3_1_0 as AdvancedPLCRVotingCheckpointChangedDefaultExemptedVotersListEventArgs,
+  AdvancedPLCRVotingCheckpointPauseEventArgs_3_1_0 as AdvancedPLCRVotingCheckpointPauseEventArgs,
+  AdvancedPLCRVotingCheckpointUnpauseEventArgs_3_1_0 as AdvancedPLCRVotingCheckpointUnpauseEventArgs,
+} from '@polymathnetwork/abi-wrappers';
+
+export type AdvancedPLCRVotingCheckpoint = AdvancedPLCRVotingCheckpoint_3_1_0;
+
+export { isAdvancedPLCRVotingCheckpoint, AdvancedPLCRVotingCheckpoint_3_1_0, isAdvancedPLCRVotingCheckpoint_3_1_0 };
+
+export namespace AdvancedPLCRVotingCheckpointTransactionParams {
```

Comment:
*All* of these exported interface names should match the name of the transaction method. So for example,

`export interface StatutoryBallot extends StatutoryBallotParams {}`

should be

`export interface CreateStatutoryBallot extends StatutoryBallotParams {}`

(Ideally, the original names in the wrapper file should also match, so `StatutoryBallotParams` -> `CreateStatutoryBallotParams`)